class.jetpack.php

```diff
@@ -468,6 +468,12 @@ private function __construct() {
 		add_action( 'init', array( $this, 'deprecated_hooks' ) );
 
 		/*
+		 * Enable enhanced handling of previewing sites in Calypso
+		 */
+		require_once JETPACK__PLUGIN_DIR . '_inc/lib/class.jetpack-iframe-embed.php';
+		add_action( 'init', array( 'Jetpack_Iframe_Embed', 'init' ), 9, 0 );
```

Comment:
@marekhrabe Unless there's a reason to, let's make sure that this code isn't loaded if Jetpack isn't connected, and I think this will be good to ship!